### ISSUE TITLE
Crash when running dispify

### ISSUE THREAD

**Issue by lwizchz** [The-Orange-Toolbox/BSPReveal#2]
Using either `-a` or `-d` doesn't work for me since dispify crashes. I'm not sure if there's anything I need to change about the map but if so I haven't. Here's the log:

```
C:\Program Files (x86)\Steam\steamapps\common\Team Fortress 2\tf\maps>"C:\Users\Louise\Downloads\CompilePal 027\Parameters\BSPReveal\BSPReveal.exe" -d cp_michael_a6b.bsp
The Orange Toolbox - BSPReveal.exe (Mar 14 2022)

Loading C:\Program Files (x86)\Steam\steamapps\common\Team Fortress 2\tf\maps\cp_michael_a6b.bsp
There is a problem with BSPReveal
Please report issues here: https://github.com/The-Orange-Toolbox/BSPReveal/issues

Traceback (most recent call last):
  File "__main__.py", line 63, in <module>
  File "dispify.py", line 143, in dispify
  File "dispify.py", line 97, in insert_lightmaps
IndexError: list index out of range
Traceback (most recent call last):
  File "__main__.py", line 63, in <module>
  File "dispify.py", line 143, in dispify
  File "dispify.py", line 97, in insert_lightmaps
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "__main__.py", line 70, in <module>
  File "totcommon\executable.py", line 25, in __exit__
  File "totcommon\reporter.py", line 18, in __exit__
NameError: name 'exit' is not defined
[2424] Failed to execute script '__main__' due to unhandled exception!
```

**Comment by maxdup** [comment]
Hi Iwizchz, I've given the code a quick look.
My understanding is that this will happen when there are no displacements featured in the map, can you confirm this is your case?

I will correct this but in the meantime, you may want to run bspreveal with 
`--clip --spawn --trigger` (every feature except disp, which you wouldn't need for this map anyway)

**Comment by lwizchz** [comment]
That's correct, there aren't any displacements yet. Thanks!

**Comment by maxdup** [comment]
Looking into the crash uncovered an other more bizarre bug that was scrambling lightmaps.

Everything should be fixed in the new version (1.3.1) https://github.com/The-Orange-Toolbox/BSPReveal/releases/tag/v1.3.1

Many thanks for reporting!
